etc/rc.init

```diff
@@ -90,7 +90,7 @@ setup_random() {
         cat /tmp/random.seed > /dev/urandom
         rm /tmp/random.seed
     else
-        dd count=1 bs=512 if=/dev/random of=/dev/urandom
+        dd count=1 bs=512 if=/dev/urandom of=/tmp/random.seed
```

Comment:
This doesn't seed the `urandom` device, which ultimately allows for early RNG prediction in the boot of the system. `urandom` also is less secure to seed from, see comment in main PR.